.bazelignore

```diff
@@ -2,4 +2,5 @@ artifacts
 bin
 build/builder_home
 lib
+pkg/ui/node_modules
```

Comment:
Adding this back because I removed it by mistake with some previous commit.

Reply:
Do the other `node_modules` folders need to be in this list as well?